tests/test_core.py

```diff
@@ -5423,6 +5423,7 @@ def test_unistd_isatty(self):
     src = open(path_from_root('tests', 'unistd', 'isatty.c')).read()
     self.do_run(src, 'success', force_c=True)
 
+  @also_with_standalone_wasm
   def test_unistd_sysconf(self):
     src = open(path_from_root('tests', 'unistd', 'sysconf.c')).read()
```

Comment:
Replace with `do_run_in_out_file()` while you are there?

Reply:
Good idea, done.